frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -368,8 +367,8 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
                         name: 'Cohorts',
                         searchPlaceholder: 'cohorts',
                         type: TaxonomicFilterGroupType.CohortsWithAllUsers,
-                        logic: cohortsModel,
-                        value: 'cohortsWithAllUsers',
+                        endpoint: combineUrl(`api/projects/${projectId}/cohorts/`).url,
+                        options: [{ id: 'all', name: 'All Users*' }],
```

Comment:
we need to include this option for `All Users` to appear as an option in the cohort search dropdown